tutorials/introductory/usage.py

```diff
@@ -299,11 +299,10 @@ def my_plotter(ax, data1, data2, param_dict):
 #
 # Without a backend explicitly set, Matplotlib automatically detects a usable
 # backend based on what is available on your system and on whether a GUI event
-# loop is already running. On Linux, if the environment variable
-# :envvar:`DISPLAY` is unset, the "event loop" is identified as "headless",
-# which causes a fallback to a noninteractive backend (agg); in all other
-# cases, an interactive backend is preferred (usually, at least tkagg will be
-# available).
+# loop is already running.  An interactive backend is usually preferred (at
+# least tkagg should be available); as an exception, on Linux, if Matplotlib
+# cannot connect to either an X display or a Wayland display, the system is
+# considered "headless" and a noninteractive backend (agg) is used.
```

Comment:
Can this be less technical and more to the point? e.g.

```suggestion
# loop is already running. The following sequence is tried: MacOSX, Qt5Agg, Gtk3Agg, TkAgg, WxAgg, Agg. The last, Agg, is a non-interactive backend that can only write to files. It is used on Linux, if Matplotlib cannot connect to either an X display or a Wayland display.
```
Sequence taken from matplotlibrc (I don't know if we check that somewhere else?

Reply:
looks good to me, took that with a very slight rewording.